pandas-stubs/_typing.pyi

```diff
@@ -195,7 +195,7 @@ IndexT = TypeVar("IndexT", bound=Index)
 
 IntervalClosedType: TypeAlias = Literal["left", "right", "both", "neither"]
 
-DateTimeErrorChoices: TypeAlias = Literal["ignore", "raise", "coerce"]
+IgnoreRaiseCoerce: TypeAlias = Literal["ignore", "raise", "coerce"]
```

Comment:
I like this change but it might be good to have a quick consensus on how we want to name type aliases:

1. Should they reflect what they are (like here `IgnoreRaiseCoerce`)
2. or should they reflect where they are used (more semantic, might leed to duplicate aliases with the same content but different names)

I think currently we mostly use 1, but I can also see the point for 2: when DateTime accepts more error codes, we only want to widen its type and not of all the other IgnoreRaiseCoerce-like error codes in different functions.

I think I slightly prefer 1, as we already went down that path.

Reply:
I prefer (1).

Reply:
The `_typing` module has quite a few literals that follow 2 - or even more generally than that - the concept, rather than where they are used

Reply:
> The reason for this change was that type was first specific to DateTime functions, but then was needed for `to_numeric`, and so needed to have a less date-time-specific name since it seemed strange to have an argument labeled DateTeimErrorChoices in `to_numeric`..

I agree with this approach.